meteor/client/ui/Settings/ConfigManifestSettings.tsx

```diff
@@ -80,6 +81,65 @@ function filterLayerMappings(
 	return result
 }
 
+function getFilteredGfxDefaultTableValues<DBInterface extends { _id: ProtectedString<any> }>(
+	item: ConfigManifestEntryFilterDefaultsFromShowMapping<boolean>,
+	configPath: string,
+	object: DBInterface,
+	alternateObject?: any
+): string[] {
+	const sourceAttribute = `${configPath}.${item.sourceTableId}`
+	const targetAttribute = `${configPath}.${item.targetTableId}`
+	const sourceTable = objectPathGet(object, sourceAttribute) ?? objectPathGet(alternateObject, sourceAttribute)
```

Comment:
Is `objectPathGet` safe to use on `alternateObject`?

Reply:
I assume this finds the table in the blueprints by looking at the object from the database. I can't see why it should differ when the blueprints exist on the same level in both objects retrieved from the database.